build.gradle

```diff
@@ -19,10 +19,12 @@ buildscript {
     repositories {
         google()
         jcenter()
+        mavenCentral()
```

Comment:
jcenter is a superset of mavenCentral and shouldn't be needed.
https://bintray.com/bintray/jcenter/me.tatarka:gradle-retrolambda